packages/firebase_auth/firebase_auth/example/android/app/google-services.json

```diff
@@ -0,0 +1,226 @@
+{
+  "project_info": {
+    "project_number": "836556442493",
+    "project_id": "manual-test-f0181",
```

Comment:
Is this file meant to be here and for this Firebase project id?

Reply:
I think I added the wrong config, will update it.

Reply:
Should be good now